companion/src/storage/firmwareinterface.cpp

```diff
@@ -220,7 +344,7 @@ void FirmwareInterface::seekSplash()
     return;
   }
 
-  if (seekSplash(QByteArray(OTX_SPS_TARANIS, OTX_SPS_SIZE), QByteArray(OTX_SPE, OTX_SPE_SIZE), 6784)) {
+  if (seekSplash(QByteArray(OTX_SPS_TARANIS, OTX_SPS_SIZE), QByteArray(OTX_SPE, OTX_SPE_SIZE), 3070)) {
```

Comment:
Can we please have a `define` for that magic number?

Reply:
There is one, but that is not shared between companion and radio.
Just tell me where to put it and I will update the PR